mypy/build.py

```diff
@@ -861,19 +865,68 @@ def is_file(path: str) -> bool:
     return res
 
 
-def find_module(id: str, lib_path_arg: Iterable[str]) -> Optional[str]:
+USER_SITE_PACKAGES = \
+    '"import site;print(site.getusersitepackages());print(*site.getsitepackages(), sep=\'\\n\')"'
+VIRTUALENV_SITE_PACKAGES = \
+    '"from distutils.sysconfig import get_python_lib;print(get_python_lib())"'
+
+
+def call_python(python: str, command: str) -> str:
+    return check_output([python, '-c', command]).decode(sys.stdout.encoding)
+
+
+def get_package_dirs(python: Optional[str]) -> List[str]:
+    """Find package directories for given python (default to Python running
```

Comment:
May I recommend PEP 257? While mypy isn't strict in its application, the idea is that a multi-line docstring should have a one-line summary, a blank line, and then more text.